### PR TITLE
Fixed technical issues with README.md

### LINKED ISSUE #23756: Fix technical issues with README.md
<em>Please make sure that this is a documentation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:doc_template</em>

**System information**
- TensorFlow version:
- Doc Link:

**Describe the documentation issue**

**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**

### PR DESCRIPTION
**Pull request by claytonjian**
Improved the README.md file to adhere to more technical writing standards. Concision was improved, errors were fixed, and formality is increased

Fixes #23756

### CODE REVIEW

**Comment by claytonjian** [comment]
I signed it!

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @drpngx: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

File: README.md [hunk 1]
```diff
@@ -12,17 +12,16 @@
 **TensorFlow** is an open source software library for numerical computation using
 data flow graphs.  The graph nodes represent mathematical operations, while
 the graph edges represent the multidimensional data arrays (tensors) that flow
-between them.  This flexible architecture enables you to deploy computation to one
-or more CPUs or GPUs in a desktop, server, or mobile device without rewriting
+between them.  This flexible architecture enables computation to CPUs or GPUs in a desktop, server, or mobile device without rewriting
```
Comment (drpngx) [inline README.md]: It's a bit subjective, but I think it's better to be less formal in an intro document. I'd undo this.

**Review by drpngx — CHANGES_REQUESTED** [review]

**Comment by ymodak** [comment]
@claytonjian Can you solve the merge conflicts to move ahead with the PR? Thanks

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by tensorflowbutler** [comment]
It has been 43 days that this pull-request has stalled. Please create a new pull-request with the requested changes.

**Pull request closed without merging**